lib/internal/Magento/Framework/Console/Cli.php

```diff
@@ -30,6 +30,7 @@
  *
  * @inheritdoc
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ * @api
```

Comment:
Usually, the `@api` annotation should be followed by other annotations, like `@SuppressWarnings`. Lets remove the `@inheritdoc` and move `@SuppressWarnings` at the end, separated by a new line.